src/TinyIoC/TinyIoC.cs

```diff
@@ -50,11 +50,8 @@
 #undef READER_WRITER_LOCK_SLIM
 #endif
 
-#if NETSTANDARD1_0 || NETSTANDARD1_1 || NETSTANDARD1_2
-#undef APPDOMAIN_GETASSEMBLIES

```

Comment:
`AppDomain` is not supported in **any** .NET Standard

Reply:
That's fine but the poly fill has been working great :)

Reply:
Can we get this reverted as the poly fill works